_posts/2020-02-14-running-a-great-socratic-seminar.md

```diff
@@ -0,0 +1,70 @@
+
+Because meetup.com is a watering hole for people interested in crypto, you can expect some natural inflow just by associating your meetup group to crypto-related keywords [LINK TO MEETUP FAQ]. Attending other crypto meetups and sharing your event with members of those groups is another easy way to increase your RSVP count. Don't be afraid of using various online communication channels to announce your event (e.g. Reddit, twitter, etc.) as well.
+
+If you're aware of specific people in a reasonable distance of your meetup who would be strong candidates for participating in discussion, invite them individually! A lot of people do not actively monitor meetup.com so the only way they might learn about your event is if you invite them. Good candidates for these invitations are individuals who are domain experts or who have very specific interests in an area that you would like to cover. Consider also the timing of your events around others in your area. If there are interesting conferences, hackathons, etc. that draw experts, try and plan your event in a way which includes them. Attendees with domain-specific knowledge can be vital to good discussion. Give them a heads up that you would like their participation--people will not naturally come prepared to engage, you have to foster that environment. 
+
+The next hurdle is content discovery and collation. The materials you select should speak to what you and the attendees will find interesting. While not every topic will be of interest to everyone, try and find things that most people can relate to. Topics we often look at include:
+- Critical vulnerabilities in popular hardware and software
+- Network statistics and phenomena (e.g. oscillations in the mempool/difficulty/UTXO set, reorgs, 51% attacks, double spends)
+- Mailing list posts with interesting proposals or critiques
+- Pull requests from popular repositories that show substantial changes in behavior or architecture as well as improve security or performance.
+- Research papers with substantial insights into topics relevant to bitcoiners
+- Blog posts that investigate topics of privacy, software development, cryptography and more. 
+
+Thankfully there are many socratic style meetups worldwide which post their content for public consumption; going through their recent events will give you a lots of materials to choose from [LINK][LINK[LINK]. Once you've collated the material you would like to cover and are preparing it for discussion, we recommend highlighting important content. Highlights are a good mnemonic device for the moderator and they help drive discussion from the crowd. Popular browser platforms have plugins that allow you to highlight copy on the page. 
```

Comment:
give you a lots of materials -> give you lots of/a lot of

Reply:
```suggestion
Thankfully there are many socratic style meetups worldwide which post their content for public consumption; going through their recent events will give you a lot of materials to choose from [LINK][LINK[LINK]. Once you've collated the material you would like to cover and are preparing it for discussion, we recommend highlighting important content. Highlights are a good mnemonic device for the moderator and they help drive discussion from the crowd. Popular browser platforms have plugins that allow you to highlight copy on the page. 
```